src/OutboundServer/handlers.js

```diff
@@ -137,6 +140,33 @@ const postAccounts = async (ctx) => {
     }
 };
 
+const postRequestToPay = async (ctx) => {
+    try {
+        // this requires a multi-stage sequence with the switch.
+        let requestToPayInboundRequest = {
```

Comment:
@ksatya77 surely this is an outbound request? ... in context of scheme adapter:
- OUTBOUND means requests from the DFSP owning the scheme adapter instance to other participants (via switch)
- INBOUND means requests coming to the scheme adapter from other participants (via switch)